frontend/src/v5/helpers/form.helper.ts

```diff
@@ -16,6 +16,7 @@
  */
```

Comment:
Category is set as required, but I don't think it's possible to select an empty option. Does it still make sense to add the asterisk there?

![image](https://github.com/3drepo/3drepo.io/assets/43547703/8b8def21-929b-4652-867b-98f0ec6ba2bc)